lifelines/fitters/coxph_fitter.py

```diff
@@ -447,13 +447,19 @@ def predict_cumulative_hazard(self, X):
         X: a (n,d) covariate numpy array or DataFrame. If a DataFrame, columns
             can be in any order. If a numpy array, columns must be in the
             same order as the training data.
-
-        Returns the cumulative hazard for the individuals.
         """
-        v = self.predict_partial_hazard(X)
-        s_0 = self.baseline_survival_
-        col = _get_index(X)
-        return pd.DataFrame(-np.dot(np.log(s_0), v.T), index=self.baseline_survival_.index, columns=col)
+        if self.strata:
```

Comment:
I've moved this logic from the predict_survival function to the more "higher up" function